orbit/pkg/update/update.go

```diff
@@ -60,6 +61,13 @@ type Options struct {
 	LocalStore client.LocalStore
 	// Targets holds the targets the Updater keeps track of.
 	Targets Targets
+	// ServerCertificate is the TLS certificate CA to use for certificate verification.
+	//
+	// If not set, then the OS CA certificate store is used.
+	ServerCertificate string
```

Comment:
File path? It should be given an suffix distinction, because it is confusing to see `ServerCertificate` and `ClientCertificate` but both are not the same type.

Reply:
Agree, addressed in 935894089.